### DISCUSSION TITLE
微博导航栏图表数字显示异常

### DISCUSSION THREAD

**Discussion by liyinred** [F9y4ng/GreasyFork-Scripts#337] [问答专区(Question / Answer)] [answered]
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/83255231/fb89be15-8780-4b8f-b182-384f7444f747)

**Comment by F9y4ng** [comment, marked as answer]
Duplicate of #42

**Reply by Tback1** [reply]
看了两遍都没找到这个问题的解决在哪儿，是脚本现在默认内涵了吗？

我自己的方案是添加 :not([class^='woo-badge-msg']) 类不渲染。全局保存，只保存weibo.com域管不了s.weibo.com等子域名。

**Reply by F9y4ng** [reply]
> 看了两遍都没找到这个问题的解决在哪儿，是脚本现在默认内涵了吗？
> 
> 我自己的方案是添加 :not([class^='woo-badge-msg']) 类不渲染。全局保存，只保存weibo.com域管不了s.weibo.com等子域名。

weibo 已内置到远程规则集中，具体使用方法参阅：#446
